python/fledge/common/configuration_manager.py

```diff
@@ -742,6 +739,8 @@ async def _update_value_val(self, category_name, item_name, new_value_val):
                 .FORMAT("return", ("ts", "YYYY-MM-DD HH24:MI:SS.MS")) \
                 .WHERE(["key", "=", category_name]).payload()
             await self._storage.update_tbl("configuration", payload)
+            cat_value = {item_name: {"value": new_value_val}}
+            self._update_special_config_items(category_name, cat_value)
```

Comment:
_set_cache_size() may be a better name or _handle_config_change

Reply:
Using such name "special" is highly discouraged. What is special?

Reply:
Special - Unlike value updates it has more things to do; that's why we have another def to tackle such things separately but NOT in the actual Update calls.

Reply:
Avoid using special like word in def name